Tribler/Test/test_metadata_community.py

```diff
@@ -1,127 +0,0 @@
-# Written by Niels Zeilemaker
-# see LICENSE.txt for license information
-
-import os
-import json
-import time
-from unittest.case import skip
-
-from Tribler.Test.test_as_server import TestGuiAsServer, TESTS_DATA_DIR
-
-from Tribler.Core.TorrentDef import TorrentDef
-from Tribler.Core.simpledefs import dlstatus_strings
-
-DEBUG = True
-
-
-class TestMetadataCommunity(TestGuiAsServer):
```

Comment:
I would keep this one, and see if you could merge it with the community stuffs. A download from an open-channel should trigger the same behaviour right?